cogs5e/models/initiative/effect.py

```diff
@@ -281,11 +283,24 @@ def parse_resist_str(resist_list):
     return ', '.join([str(Resistance.from_dict(r)) for r in resist_list])
 
 
+def parse_stat_choice(arg, _):
+    if not arg in STAT_ABBREVIATIONS:
```

Comment:
nit: `not X in Y` should be `X not in Y` (https://www.flake8rules.com/rules/E713.html)